checkita-core/src/main/scala/ru/raiffeisen/checkita/readers/ConnectionReaders.scala

```diff
@@ -107,7 +118,8 @@ object ConnectionReaders {
    * @tparam T Type of connection configuration
    */
   implicit class ConnectionReaderOps[T <: ConnectionConfig](config: T)
-                                                           (implicit reader: ConnectionReader[T]) {
+                                                           (implicit reader: ConnectionReader[T],
+                                                            spark: SparkSession) {
```

Comment:
Implicit spark session is no longer needed here since greenplum's checkConnection method was refactored.